README.md

```diff
@@ -9,7 +9,7 @@ In order to run this API locally you will need to install the following:
 - [Git](https://git-scm.com/downloads)
 
 ## Getting Started
-1. Clone this repository: `go get github.com/companieshouse/lfp-pay-api`
+1. Clone this repository: `go get github.com/companieshouse/penalty-payment-api`
```

Comment:
Don't need to revert this and we will be using this repo name and its readme.

Reply:
Ok, will leave as 'penalty-payment-api'